products/yocto.md

```diff
@@ -8,52 +8,59 @@ changelogTemplate: |
 iconSlug: NA
 releaseLabel: "__RELEASE_CYCLE__ '__CODENAME__'"
 releases:
-#  - releaseCycle: "3.5"
-#    codename: 'kirkstone'
-#    cycleShortHand: 305
-#    lts: true
-#    latest: "3.5.0"
-#    release: 2022-04-01
-#    eol:     2024-04-01
-#
+#  - releaseCycle: "4.1"
+#    codename: 'Langdale'
+#    cycleShortHand: 401
+#    latest: "4.1.0"
+#    release: 2022-10-01
+#    eol:     2023-05-01
+
+  - releaseCycle: "4.0"
+    codename: 'kirkstone'
```

Comment:
```suggestion
    codename: 'Kirkstone'
```

Reply:
Ahh, thank you, I missed that while rebasing! Also a new hardknott point release was released today! 😮